aws-cdk-gradle-git/1.91.0-gradle6.7-git.Dockerfile

```
@@ -0,0 +1,9 @@
+# justincasetech/aws-cdk:1.91.0-gradle6.7-git
+FROM justincasetech/gradle:6.7
+
+RUN set -e;\
+    apk add --no-cache nodejs=12.21.0-r0;\
+    apk add --no-cache npm=12.21.0-r0;\
```

`LTS` は `14系` になっているので、こうした方が良い気がしました

```suggestion
    apk add --no-cache nodejs=14.16.0-r0;\
    apk add --no-cache npm=14.16.0-r0;\
```

---

ご指摘ありがとうございます！

以下のようなエラーになりました。
```
ERROR: unsatisfiable constraints:
  nodejs-12.21.0-r0:
    breaks: world[nodejs=14.16.0-r0]
```

調べたところ、ベースイメージの alpine 3.21 系では node12のみ利用可能なようでした。
https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.12&arch=x86_64

さかのぼって直すの大変なので見逃して下さい〜

---

マジですか… `alpline 3.13系` でようやく切り替わるんですね
Dockerイメージがネストしていてメンテしづらいのはなんとかしたいですね… (この pull request でやることではない認識です)

https://pkgs.alpinelinux.org/packages?name=nodejs&branch=v3.13&arch=x86_64